web/packages/teleterm/src/ui/initUi.ts

```diff
@@ -51,11 +56,38 @@ function notifyAboutStoredConfigErrors(
 ): void {
   const errors = configService.getStoredConfigErrors();
   if (errors) {
+    const isKeymapError = errors.some(e =>
+      e.path[0].toString().startsWith('keymap.')
+    );
     notificationsService.notifyError({
       title: 'Encountered errors in config file',
-      description: errors
-        .map(error => `${error.path[0]}: ${error.message}`)
-        .join('\n'),
+      list: errors.map(e => `${e.path[0].toString()}: ${e.message}`),
+      description:
+        isKeymapError &&
+        'A valid shortcut contains at least one modifier and a single key code, for example "Ctrl+Shift+A".\nFunction keys do not require a modifier.',
+      link: {
+        // TODO(gzdunek): point to the properer section
+        href: 'https://goteleport.com/docs/connect-your-client/teleport-connect/',
+        text: 'See the config file documentation',
+      },
+    });
+  }
+}
+
+function notifyAboutDuplicatedShortcutsCombinations(
+  keyboardShortcutsService: KeyboardShortcutsService,
+  notificationsService: NotificationsService
+): void {
+  const duplicates = keyboardShortcutsService.getDuplicateAccelerators();
+  if (Object.keys(duplicates).length) {
+    notificationsService.notifyWarning({
```

Comment:
Any reason this is not an error? A warning will disappear automatically meaning the user might not see it as it's displayed after app launch (which takes a while), not soon after the user interacts with the app.

Reply:
I thought it can be a warning because it is not a major issue, but converting it to the error makes sense (maybe one day we should make this this auto hide feature configurable).